src/utils/Api.js

```diff
@@ -0,0 +1,87 @@
+export default class Api {
+  constructor({ baseUrl, headers }) {
+    this.baseUrl = baseUrl;
+    this._headers = headers;
+  }
+
+  getResult(res) {
```

Comment:
You've created a method for processing the sever response, so the logic is not duplicated